src/components/Search.js

```diff
@@ -1,159 +1,209 @@
-        )}
+    useEffect(() => {
+        const { skills, tags } = queryParams;
+        (!isEmpty(skills) || !isEmpty(tags)) && handleSearch();
+    }, [queryParams]);
 
-        {/* Tags card */}
-        {!!tags?.length && (
-          <div className='skill-tag-card'>
-            <h2>Tags</h2>
-            {tags?.map((tag, index) => (
-              <div key={`tag-${index}`} className='project-form-element'>
-                <input
-                  id='tags'
-                  name={tag}
-                  type='checkbox'
-                  onChange={handleInputChange}
-                />
-                <label htmlFor={tag}>{tag}</label>
-              </div>
-            ))}
-          </div>
-        )}
-      </div>
+    useEffect(() => {
+        username &&
+            axiosInstance.get(`/users/${username}`).then((res) => {
+                setUserSkills(res?.data?.skills);
+            });
+    }, [username]);
 
-      {/* Search and projects */}
-      <div className='search-projects'>
-        <div className='searchbar'>
-          <input
-            type='text'
-            value={searchQuery}
-            className='form-control'
-            id='search'
-            placeholder='Search...'
-            onChange={({ target: { value } }) => setSearchQuery(value)}
-          />
-          <button className='btn btn-primary' onClick={handleSearch}>
-            Serach
-          </button>
-        </div>
-        {!!projects?.length && (
-          <div className='project-list'>
-            {projects?.map((project) => {
-              const { id, name, description, creator, skills, tags } = project;
-              return (
-                <div
-                  key={`project-${id}`}
-                  className='project-card'
-                  onClick={() => navigateToProject(id)}
-                >
-                  <div className='custom-card-header'>
-                    <h3>{name}</h3>
-                    <h5>{creator}</h5>
-                  </div>
-                  <div className='custom-card-body'>
-                    <h4>{description}</h4>
-                  </div>
-                  <div className='custom-card-footer'>
-                    <div className='skill-tags-detail'>
-                      <h6>Skills</h6>
-                      <p>{skills.join(', ')}</p>
+    const fetchMoreData = () => {
+        if (loadedRecords.length < projects.length) {
+            setLoadedRecords(loadedRecords.concat(projects.splice(0, 30)));
+        } else {
+            setHasMore(false);
+        }
+    };
+    return (
+        <div className="search-page">
+            <div className="skills-tag-container">
+                {/* Skills card */}
+                {!!skills?.length && (
+                    <div className="skill-tag-card">
+                        <h2>Skills</h2>
+                        {skills?.map((skill, index) => (
+                            <div key={`tag-${index}`} className="project-form-element">
```

Comment:
This is a skill, so shouldn't it be `skill-${index}`? It might interfere with the tags rendering, too.